CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
 ## Unreleased
-* Added a new validation that checks whether a pack should be hidden.
-* Added a new format to `pack_metadata.json` to hide a pack that should be hidden.
+* Fixed an issue where the **validate** command sometimes returned a false negative in cases where there are several sub-playbooks with the same ID.
```

Comment:
is this relevant to this pr?

Reply:
not sure why you see it as a change, i don't see it as a change, maybe you didn't refresh the files?

Reply:
youre right, not green anymore